onnxruntime/core/providers/shared_library/provider_api.h

```diff
@@ -436,6 +436,8 @@ inline Env& GetDefaultEnv() {
 }
 }  // namespace onnxruntime
 
+size_t MLASCALL MlasGetPreferredBufferAlignment();
```

Comment:
What provider bridge EP requires this? I don't see any usage in the plugin ep changes.

Reply:
It is implicit from OrtDevice ctor.
Said that, We will let EP set that at the point of construction.